yb-voyager/src/srcdb/postgres.go

```diff
@@ -80,8 +80,7 @@ func (pg *PostgreSQL) GetVersion() string {
 	}
 	return version
 }
-
-func (pg *PostgreSQL) GetAllTableNames() []string {
+func checkSchemasExists(pg *PostgreSQL) []string{
```

Comment:
Can't we make this as a method of `PostgreSQL` type?

Leave one empty line above this function definition.